docker-entrypoint.sh

```diff
@@ -23,5 +23,7 @@ do
   fi
 done
 
+# TODO: Can remove the next line when DAC moves to kubectl >= 1.26
```

Comment:
This release is to support kubernetes 1.26. I would suggest saying that this will be removed when the default value is moved to 1.26 and greater.

Reply:
I applied your suggestion.